src/pages/LVL_LevelEdit.page

```diff
@@ -53,8 +53,12 @@
                                     <apex:actionStatus id="statusRefill"
                                         onstart="document.getElementById('divLoading').style.display = 'inline';"
                                         onstop="document.getElementById('divLoading').style.display = 'none';" />
-                                    <div class="slds-spinner_small mySpinner" id="divLoading" >
-                                        <img src="{!URLFOR($Asset.SLDS, '/assets/images/spinners/slds_spinner_brand.gif')}" alt="{!$Label.labelMessageLoading}" />
+                                    <div id="divLoading" class="slds-spinner_container slds-is-relative mySpinner" >
```

Comment:
Spinner is slightly outside the field now, not sure that's a big deal but figured I'd mention it.

Reply:
@force2b did you have a chance to look into the spinner alignment issue?

Reply:
Yes, but I never actually fixed it until now.